content/cybersecurity/concepts/nmap/terms/tcp-syn-scan/tcp-syn-scan.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'TCP SYN Scan'
+Description: 'Performs a scan that can indicate open, filtered, and closed port states with a reduced risk of detection, since it never completes a full TCP connection.'
+Subjects:
+  - 'Networking'
+  - 'Cybersecurity'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'introduction-to-ethical-hacking'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP SYN scan** runs by default when running Nmap as the root user or Administrator. It is the most popular scan option according to [Nmap.org](https://nmap.org/book/synscan.html). Quick and efficient, this scan can indicate open, filtered, and closed port states. It is requested by passing the `-sS` option to Nmap, but when running as the root user or Administrator, `-sS` is usually omitted. Also known as the half-open scan, it never completes the full TCP connection, so it's less likely to be blocked by firewalls.
+
+## Syntax
+
+Root privileges are required to perform a TCP SYN scan and the target can be any system with an established network connection. The following command performs a scan on `<target>` after entering the user password when prompted:
```

Comment:
I'm wondering if this is a fact that we need root privileges. It looks like it's more so a matter of having permissions to send/receive raw-packets (see https://nmap.org/book/synscan.html).

Reply:
You are right, it is all about raw-packets. But then you need root privileges to send/receive them.

Reply:
I see! In that case, can we take the second sentence

> It is requested by passing the `-sS` option to Nmap, but when running as the root user or Administrator, `-sS` is usually omitted. Also known as the half-open scan, it never completes the full TCP connection, so it's less likely to be blocked by firewalls.

and move it to the syntax section you think it best fits? Maybe we can replace the note below with that information somewhat worded differently and referencing the pseudo block.

Reply:
@yangc95 I made the changes as you suggested. Please, check if it's okay.